routes/web.php

```diff
@@ -35,13 +35,13 @@
 		$r->post( '/register', [ UserController::class, 'register_process' ] );
 	} );
 
-	$r->middleware( 'magic:launcher/app', function ( Routes $r ) {
-		$r->group( 'launcher/app/{key}', function ( Routes $r ) {
-			$r->get( '', [ HomeController::class, 'show' ] );
-			$r->get( '/subpage', [ SubpageController::class, 'show' ] );
-			$r->get( '/{path:.*}', fn( Request $request, Response $response ) => $response->setStatusCode( 404 ) );
-		} );
-	} );
+    $r->middleware( [ 'magic:launcher/app', 'dt_launcher_share' ], function ( Routes $r ) {
```

Comment:
@ancy2023 , please use `check_share` as your middleware value rather than the cookie name to match the config set in the MiddlewareServiceProvider route middleware array.